workflows-scripts/users/users-loan-renewal/jmeter-supported-data/loans.tsv

```diff
@@ -0,0 +1,10 @@
+9ded16fa-6e11-4c53-89c1-a76d62482d80	{"id": "9ded16fa-6e11-4c53-89c1-a76d62482d80", "action": "checkedout", "itemId": "3faad467-fca1-41c8-ad7d-af1dcd3fdc3c", "status": {"name": "Open"}, "userId": "4deba809-4fb4-432a-b3df-c54b6ad51d22", "dueDate": "2019-09-06T07:22:30.000+0000", "loanDate": "2020-05-11T23:08:27.297068+0000", "metadata": {"createdDate": "2020-05-11T23:08:30.31", "updatedDate": "2020-05-11T23:08:30.802+0000", "createdByUserId": "9a4e9b21-8404-437e-abb2-76998218ab73", "updatedByUserId": "9a4e9b21-8404-437e-abb2-76998218ab73"}, "itemStatus": "Checked out", "loanPolicyId": "2be97fb5-eb89-46b3-a8b4-776cea57a99e", "lostItemPolicyId": "6699ab75-d639-4dd4-9a82-b3ab0ffffd0a", "overdueFinePolicyId": "0f8193f2-8338-4def-b53d-893acaa6f20a", "checkoutServicePointId": "7068e104-aa14-4f30-a8bf-71f71cc15e07", "patronGroupIdAtCheckout": "648ec9f2-c292-4e74-932b-9e5cf059d4d6"}
```

Comment:
Is there a plan to add more loans to test the scenarios of 10+ loans. If not how can these 10 loans be used in the scenarios requiring more than 10 loans?

Reply:
I have already tested with 10, 50, 100,200 loans for a user. I did not checkin file with 200 loans because it unnecessary increases repository size. We can easily generate 200+ loans using the sql query mentioned in Readme.